apps/condo/domains/common/components/FocusContainer.ts

```diff
@@ -1,9 +1,18 @@
 import styled from '@emotion/styled'
 import { colors } from '../constants/style'
 
-export const FocusContainer = styled.div`
+export const FocusContainer = styled.div<{
```

Comment:
Why did focus container know smthng about mode and color?  Why did u change padding, this changes can break markup at ticket/[index].tsx